.maintain/chaostest/src/hypervisor/modules/k8s.js

```diff
@@ -0,0 +1,113 @@
+const k8s = require('@kubernetes/client-node')
+const { isFunction } = require('../../utils')
+const logger = require('../../utils/logger')
+
+// load k8s
+const kc = new k8s.KubeConfig()
+kc.loadFromDefault()
+
+// load k8s Apis
+const k8sAppApi = kc.makeApiClient(k8s.AppsV1Api)
+const k8sCoreApi = kc.makeApiClient(k8s.CoreV1Api)
+
+const createNameSpace = async namespace => {
```

Comment:
Nit pick: 

```suggestion
const createNamespace = async namespace => {
```

No need to treat namespace as two words (see https://en.wikipedia.org/wiki/Namespace).

Reply:
resolved